### PR TITLE
A lot of touch up from Karsten Engsager

### PR DESCRIPTION
**Pull request by busstoptaktik**
(WIP, but much improvement already)

### CODE REVIEW

File: resources/DK.ke0 [hunk 1]
```diff
@@ -0,0 +1,142 @@
+################################################################################
```
Comment (kbevers) [inline resources/DK.ke0]: This file is not meant to be part of the distribution, right?

File: resources/DK.kreve [hunk 2]
```diff
@@ -0,0 +1,75 @@
+################################################################################
```
Comment (kbevers) [inline resources/DK.kreve]: Same as above

File: resources/FO.ke0 [hunk 3]
```diff
@@ -0,0 +1,57 @@
+###############################################################################
+#
```
Comment (kbevers) [inline resources/FO.ke0]: Same as above

File: resources/DK.kreve (hunk 2 above)
Comment (busstoptaktik) [reply resources/DK.kreve]: No - not a part of the final distribution, but necessary to share right now, to get KE synchronized

File: resources/DK.ke0 (hunk 1 above)
Comment (busstoptaktik) [reply resources/DK.ke0]: (same as below)

File: resources/DK.kreve (hunk 2 above)
Comment (kbevers) [reply resources/DK.kreve]: Alright. Just wanted to make sure this don't get merged accidentally.

**Comment by busstoptaktik** [comment]
Nope - another cleaned up pull request comming...

2017-11-09 15:08 GMT+01:00 Kristian Evers <notifications@github.com>:

**Comment by busstoptaktik** [comment]
Superceded by #4

**Pull request closed without merging**